src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,40 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage {
+    @FindBy(css="img.avatar" )
+    WebElement userAvatarIcon;
+    public HomePage(WebDriver givenDriver) {
+        super (givenDriver);
+
+    }
+    public WebElement getUserAvatar(){
+        return wait.until(ExpectedConditions.visibilityOfElementLocated((By) userAvatarIcon));
+    }
+    @FindBy(css=".playlist:nth-child(3)")
+    WebElement firstPlaylist;
+    @FindBy(css="input[name='name']")
+    WebElement playlistInputField;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement use
```